packages/dev-scripts/bin/sf-prepack.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const shell = require('../utils/shelljs');
+const { isPlugin } = require('../utils/project-type');
+const packageRoot = require('../utils/package-path');
+
+shell.exec('yarn build');
+
+if (isPlugin(packageRoot)) {
+  shell.exec('oclif-dev manifest');
```

Comment:
If this fails, should it stop the entire process or keeping going?  As written, I don't think it would.

over on testkit, we've been having to inspect the output of shell.exec (because you can't use the `fatal` option because of a bug) to verify success.